src/Microsoft.OData.Core/Microsoft.OData.Core.cs

```diff
@@ -193,7 +193,9 @@ internal sealed class TextRes {
         internal const string ODataBatchReaderStream_MultiByteEncodingsNotSupported = "ODataBatchReaderStream_MultiByteEncodingsNotSupported";
         internal const string ODataBatchReaderStream_UnexpectedEndOfInput = "ODataBatchReaderStream_UnexpectedEndOfInput";
         internal const string ODataBatchReaderStreamBuffer_BoundaryLineSecurityLimitReached = "ODataBatchReaderStreamBuffer_BoundaryLineSecurityLimitReached";
+        internal const string ODataJsonLightBatchPayloadItemPropertiesCache_DuplicatePropertyForMessageInBatch = "ODataJsonLightBatchPayloadItemPropertiesCache_DuplicatePropertyForMessageInBatch";

```

Comment:
>ODataJsonLightBatchPayloadItemPropertiesCache_DuplicatePropertyForMessageInBatch [](http://example.com/codeflow?start=114&length=80)

These error strings need to be added to Microsoft.OData.Core.txt.

Reply:
Added both exceptions into the Microsoft.OData.Core.txt